editor/resources/bundle.editor_script

```diff
@@ -122,7 +122,8 @@ local function find_existing_path(paths)
             for j = 1, #path do args[#args+1] = path[j] end
             args[#args+1] = {reload_resources = false, out = "capture"}
             local success, result = pcall(editor.execute, table.unpack(args))
-            resolved_path = success and result
+            -- take the first line in case e.g. `where` returns multiple matches
+            resolved_path = success and result and (result:match("^(.-)\r?\n") or result)
```

Comment:
This fixes the following error from [the post](https://forum.defold.com/t/defold-1-9-8-beta/79816/26?u=vlaaad):
```
Resolving ADB location...
ERROR:EXT: bundle.editor_script:132 vm error: java.nio.file.InvalidPathException: Illegal char <
> at index 59: C:\Users\U\AppData\Local\Android\sdk\platform-tools\adb.exe
ERROR:EXT: C:\Users\U\adb\adb.exe
```
It's not immediately apparent from the error message, but it complains that there is an unexpected newline character in path, so the resolved adb path was something like `"C:\\...\\platform-tools\\adb.exe\r\nC:\\...\\adb\\adb.exe"`. This is because Windows' `where` command can actually return multiple results, each on a separate line, which I didn't know when I coded this initially.